### PR TITLE
tests: create basic test for creating VMs on github actions

### PR DESCRIPTION
**Pull request by justinsb**
We try to bring up 3 VMs and SSH to them.

### CODE REVIEW

**Comment by justinsb** [comment]
/wip

@assign @hakman 

I figure _if_ we can run KVM/qemu on github actions, that would be a nice way to test.  Still WIP because I need to hook this up to github actions!

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -80,3 +80,20 @@ jobs:
         working-directory: ${{ env.GOPATH }}/src/k8s.io/kops
         run: |
           make quick-ci
+
+  tests-e2e-scenarios-bare-metal:
```
Comment (hakman) [inline .github/workflows/main.yml:84]: Should we move the test to a different file?

Comment (justinsb) [reply .github/workflows/main.yml:84]: I was debating it ... yes, I think it's a good idea!  `presubmits.yaml` ?  `e2e.yaml` ?

Comment (hakman) [reply .github/workflows/main.yml:84]: `e2e` or `metal`. We can always rename later.

**Comment by justinsb** [comment]
/retest

A few pod scheduling timeouts on prow, should not be related to this or even to kOps code..

File: .github/workflows/main.yml (hunk 1 above)
Comment (justinsb) [reply .github/workflows/main.yml]: e2e works for me - thanks!

**Review by hakman — APPROVED** [review]

**Pull request merged**